### PR TITLE
Build cache update

### PR DESCRIPTION
**Pull request by FeodorFitsner**

### CODE REVIEW

**Comment by XhmikosR** [comment]
@FeodorFitsner: I rebased the staging branch. Please always rebase it against master instead of doing merges to avoid issues in the future.

File: Gemfile.lock [hunk 1]
```diff
@@ -51,6 +52,7 @@ GEM
 
 PLATFORMS
   x64-mingw32
+  x86-mingw32
```
Comment (XhmikosR) [inline Gemfile.lock]: This isn't right; it has unrelated commits.

File: dev-build.cmd [hunk 2]
```diff
@@ -0,0 +1,2 @@
+call grunt build
```
Comment (XhmikosR) [inline dev-build.cmd]: Why do you need this patch at all?

You test it locally with IIS? If so, you should change the first line to `cmd /c` instead of call.

**Comment by XhmikosR** [comment]
See @FeodorFitsner the merge patches should not be here. It's wrong.

This patch, https://github.com/appveyor/website/pull/199/commits/c3e5f947b6c632560085af7550da9e874bcca747, is already in staging and master.

**Comment by FeodorFitsner** [comment]
What do you mean?

**Comment by FeodorFitsner** [comment]
I just did "Sync" in GitHub Desktop - nothing unusual.

**Comment by XhmikosR** [comment]
You forced pushed. You shouldn't have **unless* you understand what you are doing.

**Comment by XhmikosR** [comment]
> I just did "Sync" in GitHub Desktop - nothing unusual.

There's your problem. `git fetch && git reset --hard` along with the other things we discussed on Skype.
